action.yml

```diff
@@ -45,11 +45,20 @@ inputs:
       max-age parameter for the aws sync command when copying files for version deploys.
       This overrides the built-in value for version deploys.
     required: false
+  github-token:
+    description: GitHub token
+    required: true
+  deployRunUrl:
+    description: "Templated string for the URL to run the deployed files. The string can
+      contain the following placeholder: __deployPath__"
+    required: true
```

Comment:
This shouldn't be required as described below. Otherwise repos using this action will break when the new version is released.